HFStrategy/Strategy/Order.py

```diff
@@ -0,0 +1,14 @@
+import time

```

Comment:
I briefly looked at how the Order object is strutured `bitfinex-api-node` and ran away in fear (its big). Is the plan to copy this as close as possible?

https://github.com/bitfinexcom/bitfinex-api-node/blob/v2.0.0/lib/models/order.js

Reply:
No, it is overkill for this repo I think. It's meant for general usage with the Bitfinex API. We should copy over the flag methods and accessors (quote/base currencies)